### ISSUE TITLE
Basket Number

### ISSUE THREAD

**Issue by kellymc03** [thekesolutions/koha-plugin-gobi#21]
Gobi would like a response from back Koha to ensure the file was received and processed.  Order numbers are assigned by Koha to each record it receives in the Gobi file.  If this file consisted of multiple records it would be more useful for Koha to send back the basket number to Gobi instead of multiple order numbers.

**Comment by tomascohen** [comment]
I agree: we should be returning the basket number in all cases. This is due to a terminology discrepancy between Koha and (most) acquisitions management systems: baskets vs. orders.
